docs/mllib-clustering.md

```diff
@@ -593,6 +593,58 @@ ssc.start()
 ssc.awaitTermination()
 
 {% endhighlight %}
+</div>
+
+<div data-lang="python" markdown="1">
+First we import the neccessary classes.
+
+{% highlight python %}
+
+from pyspark.mllib.linalg import Vectors
+from pyspark.mllib.regression import LabeledPoint
+from pyspark.mllib.clustering import StreamingKMeans
+
+{% endhighlight %}
+
+Then we make an input stream of vectors for training, as well as a stream of labeled data
+points for testing. We assume a StreamingContext `ssc` has been created, see
+[Spark Streaming Programming Guide](streaming-programming-guide.html#initializing) for more info.
+
+{% highlight python %}
+
+trainingData = ssc.textFileStream("/training/data/dir").map(Vectors.parse)
+testData = ssc.textFileStream("/testing/data/dir").map(LabeledPoint.parse)
```

Comment:
@mengxr Should I remove `LabeledPoint.parse` and add my own parse function for this example, since loadLabeledPoints outputs a RDD but we want a mapped dstream . Would be great to post the final consensus here https://github.com/apache/spark/pull/6746

cc @freeman-lab